### DISCUSSION TITLE
Add application changelog for helm chart updates

### DISCUSSION THREAD

**Discussion by treezio** [updatecli/updatecli#1835] [Q&A]
We are currently managing the application upgrades using the helm chart version.
As the changelog for helm charts most of the times does not contain really valuable information about the application that it manages, I would like to include some information about the application changelog in the Pull request created by updatecli.

I though it could be nice to define some pipeline as:

1. Get chart source
2. Parse appVersion value (application version) from Chart.yaml
3. Retrieve application information using Github Release Source filtering by appVersion parsed in step 2
4. Create pull request containing information from both chart and application release.

I've tried some approaches using `githubrelease` with semver and regex filter with no luck.

Is this something that updatecli can handle ?

Thanks !

**Comment by olblak** [comment]
It would definitely make sense, I did something similar for other plugins like golang 

-> https://github.com/updatecli/updatecli/blob/main/pkg/plugins/resources/go/module/changelog.go

I had a quick look to https://helm.sh/docs/topics/charts/
and we should be able to leverage sources and appVersion even though there is no convention with the content of sources

I think the helm changelog should be updated [here](https://github.com/updatecli/updatecli/blob/main/pkg/plugins/resources/helm/changelog.go)
